source/renderer/app/features/discreet-mode/feature.ts

```diff
@@ -50,6 +51,12 @@ export class DiscreetMode extends Feature {
   @action
   toggleDiscreetMode = () => {
     this.isDiscreetMode = !this.isDiscreetMode;
+    this.analyticsTracker.sendEvent(
+      EventCategories.SETTINGS,
+      this.isDiscreetMode
+        ? 'Turned on discreet mode'
```

Comment:
Should we create some constants for the event names? Sounds fair, easier to maintain and less prone to typos.

Reply:
I've already discussed it with Shawn: https://github.com/input-output-hk/daedalus/pull/3003#discussion_r907583452

Let me know if you'd want me to introduce constants for event names anyway.

Reply:
As discussed offline 🙂 In future we might need event versioning or something else but for now let's keep it YAGNI way. Thanks